OgreMain/src/Compositor/OgreCompositorNode.cpp

```diff
@@ -868,7 +868,9 @@ namespace Ogre
 
             const CompositorTargetDef *targetDef = passDef->getParentTargetDef();
 
-            if( executionMask & passDef->mExecutionMask &&
+            uint32 combinedMask = executionMask & passDef->mExecutionMask;
+
+            if( ( combinedMask & ~ExecutionFlags::RESERVED_EXECUTION_FLAGS ) && ( combinedMask & ExecutionFlags::RESERVED_EXECUTION_FLAGS ) &&
```

Comment:
Split the execution mask test in two step: bits used internally by OGRE and bits reserved to the user